pkgs/development/python-modules/python3-application/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, isPy3k, buildPythonPackage, fetchFromGitHub, fetchpatch, zope_interface, twisted }:
+
+buildPythonPackage rec {
+  pname = "python3-application";
+  version = "3.0.3";
+
+  disabled = !isPy3k;
```

Comment:
This should use pythonOlder instead.

Reply:
I think it is appropriate to use `isPy3k` for this package, because python3 is in the name.